server/src/main/java/org/loomdev/loom/entity/player/PlayerImpl.java

```diff
@@ -175,12 +174,13 @@ public int getProtocolVersion() {
     @Override
     @NotNull
     public Optional<Component> getTabListName() {
-        return Optional.ofNullable(tabListName);
+        return Optional.ofNullable(TextTransformer.toLoom(getMinecraftEntity().tabListName));
```

Comment:
Same deal as earlier; just move the variable to this class. I don't see a need to have it in the Minecraft code. Text transformation can be moved to the NMS method.

Same thing goes for `setTabListName`.